mojap_metadata/extractors/connect.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import create_engine
+
+
+def create_postgres_connection(db_settings, database_name="postgres"):
```

Comment:
It would be good to add type hints here for these functions (once they have been moved based on the suggestions from @s-ducks )

Reply:
Done